assignment/impl/src/java/org/sakaiproject/assignment/impl/persistence/AssignmentRepositoryImpl.java

```diff
@@ -139,6 +162,11 @@ public AssignmentSubmission findSubmission(String submissionId) {
         return (AssignmentSubmission) sessionFactory.getCurrentSession().get(AssignmentSubmission.class, submissionId);
     }
 
+    @Override
+    public AssignmentTimeSheet findTimeSheet(Long timeSheetId) {
+        return (AssignmentTimeSheet) sessionFactory.getCurrentSession().get(AssignmentTimeSheet.class, timeSheetId);
```

Comment:
make sessionFactory.getCurrentSession() a local var since it's used more than once

Reply:
Moved to class declaration and updated every instance.